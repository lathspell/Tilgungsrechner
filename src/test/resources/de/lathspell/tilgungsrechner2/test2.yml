## Tilgungsrechner2 Konfiguration
---
months:
    "2016-04":
        auszahlung: 120000
        sollzins: 2.09
        monatsrate: 800
    "2020-01":
        sondertilgung: 10000
    "2024-01":
        monatsrate: 1009
    