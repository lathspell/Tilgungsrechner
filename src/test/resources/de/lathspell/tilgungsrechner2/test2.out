# test2
Tilgungsplan
Periode 	Rate (EUR) 	Zinsanteil (EUR) 	Tilgungsanteil (EUR) 	Restschuld am Periodenende (EUR)
Apr 2016           800,00             209,00                 591,00                119.409,00
Mai 2016           800,00             207,97                 592,03                118.816,97
Jun 2016           800,00             206,94                 593,06                118.223,91
Jul 2016           800,00             205,91                 594,09                117.629,82
Aug 2016           800,00             204,87                 595,13                117.034,69
Sep 2016           800,00             203,84                 596,16                116.438,53
Okt 2016           800,00             202,80                 597,20                115.841,33
Nov 2016           800,00             201,76                 598,24                115.243,09
Dez 2016           800,00             200,72                 599,28                114.643,81
Jan 2017           800,00             199,67                 600,33                114.043,48
Feb 2017           800,00             198,63                 601,37                113.442,11
Mär 2017           800,00             197,58                 602,42                112.839,69
Apr 2017           800,00             196,53                 603,47                112.236,22
Mai 2017           800,00             195,48                 604,52                111.631,70
Jun 2017           800,00             194,43                 605,57                111.026,13
Jul 2017           800,00             193,37                 606,63                110.419,50
Aug 2017           800,00             192,31                 607,69                109.811,81
Sep 2017           800,00             191,26                 608,74                109.203,07
Okt 2017           800,00             190,20                 609,80                108.593,27
Nov 2017           800,00             189,13                 610,87                107.982,40
Dez 2017           800,00             188,07                 611,93                107.370,47
Jan 2018           800,00             187,00                 613,00                106.757,47
Feb 2018           800,00             185,94                 614,06                106.143,41
Mär 2018           800,00             184,87                 615,13                105.528,28
Apr 2018           800,00             183,80                 616,20                104.912,08
Mai 2018           800,00             182,72                 617,28                104.294,80
Jun 2018           800,00             181,65                 618,35                103.676,45
Jul 2018           800,00             180,57                 619,43                103.057,02
Aug 2018           800,00             179,49                 620,51                102.436,51
Sep 2018           800,00             178,41                 621,59                101.814,92
Okt 2018           800,00             177,33                 622,67                101.192,25
Nov 2018           800,00             176,24                 623,76                100.568,49
Dez 2018           800,00             175,16                 624,84                 99.943,65
Jan 2019           800,00             174,07                 625,93                 99.317,72
Feb 2019           800,00             172,98                 627,02                 98.690,70
Mär 2019           800,00             171,89                 628,11                 98.062,59
Apr 2019           800,00             170,79                 629,21                 97.433,38
Mai 2019           800,00             169,70                 630,30                 96.803,08
Jun 2019           800,00             168,60                 631,40                 96.171,68
Jul 2019           800,00             167,50                 632,50                 95.539,18
Aug 2019           800,00             166,40                 633,60                 94.905,58
Sep 2019           800,00             165,29                 634,71                 94.270,87
Okt 2019           800,00             164,19                 635,81                 93.635,06
Nov 2019           800,00             163,08                 636,92                 92.998,14
Dez 2019           800,00             161,97                 638,03                 92.360,11
>> Sondertilgung 10.000,00 EUR nach Periode Dez 2019
Jan 2020           800,00             143,44                 656,56                 81.703,55
Feb 2020           800,00             142,30                 657,70                 81.045,85
Mär 2020           800,00             141,15                 658,85                 80.387,00
Apr 2020           800,00             140,01                 659,99                 79.727,01
Mai 2020           800,00             138,86                 661,14                 79.065,87
Jun 2020           800,00             137,71                 662,29                 78.403,58
Jul 2020           800,00             136,55                 663,45                 77.740,13
Aug 2020           800,00             135,40                 664,60                 77.075,53
Sep 2020           800,00             134,24                 665,76                 76.409,77
Okt 2020           800,00             133,08                 666,92                 75.742,85
Nov 2020           800,00             131,92                 668,08                 75.074,77
Dez 2020           800,00             130,76                 669,24                 74.405,53
Jan 2021           800,00             129,59                 670,41                 73.735,12
Feb 2021           800,00             128,42                 671,58                 73.063,54
Mär 2021           800,00             127,25                 672,75                 72.390,79
Apr 2021           800,00             126,08                 673,92                 71.716,87
Mai 2021           800,00             124,91                 675,09                 71.041,78
Jun 2021           800,00             123,73                 676,27                 70.365,51
Jul 2021           800,00             122,55                 677,45                 69.688,06
Aug 2021           800,00             121,37                 678,63                 69.009,43
Sep 2021           800,00             120,19                 679,81                 68.329,62
Okt 2021           800,00             119,01                 680,99                 67.648,63
Nov 2021           800,00             117,82                 682,18                 66.966,45
Dez 2021           800,00             116,63                 683,37                 66.283,08
Jan 2022           800,00             115,44                 684,56                 65.598,52
Feb 2022           800,00             114,25                 685,75                 64.912,77
Mär 2022           800,00             113,06                 686,94                 64.225,83
Apr 2022           800,00             111,86                 688,14                 63.537,69
Mai 2022           800,00             110,66                 689,34                 62.848,35
Jun 2022           800,00             109,46                 690,54                 62.157,81
Jul 2022           800,00             108,26                 691,74                 61.466,07
Aug 2022           800,00             107,05                 692,95                 60.773,12
Sep 2022           800,00             105,85                 694,15                 60.078,97
Okt 2022           800,00             104,64                 695,36                 59.383,61
Nov 2022           800,00             103,43                 696,57                 58.687,04
Dez 2022           800,00             102,21                 697,79                 57.989,25
Jan 2023           800,00             101,00                 699,00                 57.290,25
Feb 2023           800,00              99,78                 700,22                 56.590,03
Mär 2023           800,00              98,56                 701,44                 55.888,59
Apr 2023           800,00              97,34                 702,66                 55.185,93
Mai 2023           800,00              96,12                 703,88                 54.482,05
Jun 2023           800,00              94,89                 705,11                 53.776,94
Jul 2023           800,00              93,66                 706,34                 53.070,60
Aug 2023           800,00              92,43                 707,57                 52.363,03
Sep 2023           800,00              91,20                 708,80                 51.654,23
Okt 2023           800,00              89,96                 710,04                 50.944,19
Nov 2023           800,00              88,73                 711,27                 50.232,92
Dez 2023           800,00              87,49                 712,51                 49.520,41
>> Neuer Tilgungssatz 8,00 % ab Jan 2024 (neue Rate 1.009,00)
Jan 2024         1.009,00              86,25                 922,75                 48.597,66
Feb 2024         1.009,00              84,64                 924,36                 47.673,30
Mär 2024         1.009,00              83,03                 925,97                 46.747,33
Apr 2024         1.009,00              81,42                 927,58                 45.819,75
Mai 2024         1.009,00              79,80                 929,20                 44.890,55
Jun 2024         1.009,00              78,18                 930,82                 43.959,73
Jul 2024         1.009,00              76,56                 932,44                 43.027,29
Aug 2024         1.009,00              74,94                 934,06                 42.093,23
Sep 2024         1.009,00              73,31                 935,69                 41.157,54
Okt 2024         1.009,00              71,68                 937,32                 40.220,22
Nov 2024         1.009,00              70,05                 938,95                 39.281,27
Dez 2024         1.009,00              68,41                 940,59                 38.340,68
Jan 2025         1.009,00              66,78                 942,22                 37.398,46
Feb 2025         1.009,00              65,14                 943,86                 36.454,60
Mär 2025         1.009,00              63,49                 945,51                 35.509,09
Apr 2025         1.009,00              61,84                 947,16                 34.561,93
Mai 2025         1.009,00              60,20                 948,80                 33.613,13
Jun 2025         1.009,00              58,54                 950,46                 32.662,67
Jul 2025         1.009,00              56,89                 952,11                 31.710,56
Aug 2025         1.009,00              55,23                 953,77                 30.756,79
Sep 2025         1.009,00              53,57                 955,43                 29.801,36
Okt 2025         1.009,00              51,90                 957,10                 28.844,26
Nov 2025         1.009,00              50,24                 958,76                 27.885,50
Dez 2025         1.009,00              48,57                 960,43                 26.925,07
Jan 2026         1.009,00              46,89                 962,11                 25.962,96
Feb 2026         1.009,00              45,22                 963,78                 24.999,18
Mär 2026         1.009,00              43,54                 965,46                 24.033,72
Apr 2026         1.009,00              41,86                 967,14                 23.066,58
Mai 2026         1.009,00              40,17                 968,83                 22.097,75
Jun 2026         1.009,00              38,49                 970,51                 21.127,24
Jul 2026         1.009,00              36,80                 972,20                 20.155,04
Aug 2026         1.009,00              35,10                 973,90                 19.181,14
Sep 2026         1.009,00              33,41                 975,59                 18.205,55
Okt 2026         1.009,00              31,71                 977,29                 17.228,26
Nov 2026         1.009,00              30,01                 978,99                 16.249,27
Dez 2026         1.009,00              28,30                 980,70                 15.268,57
Jan 2027         1.009,00              26,59                 982,41                 14.286,16
Feb 2027         1.009,00              24,88                 984,12                 13.302,04
Mär 2027         1.009,00              23,17                 985,83                 12.316,21
Apr 2027         1.009,00              21,45                 987,55                 11.328,66
Mai 2027         1.009,00              19,73                 989,27                 10.339,39
Jun 2027         1.009,00              18,01                 990,99                  9.348,40
Jul 2027         1.009,00              16,28                 992,72                  8.355,68
Aug 2027         1.009,00              14,55                 994,45                  7.361,23
Sep 2027         1.009,00              12,82                 996,18                  6.365,05
Okt 2027         1.009,00              11,09                 997,91                  5.367,14
Nov 2027         1.009,00               9,35                 999,65                  4.367,49
Dez 2027         1.009,00               7,61               1.001,39                  3.366,10
Jan 2028         1.009,00               5,86               1.003,14                  2.362,96
Feb 2028         1.009,00               4,12               1.004,88                  1.358,08
Mär 2028         1.009,00               2,37               1.006,63                    351,45
Apr 2028           352,06               0,61                 351,45                      0,00
